charts/merlin/values.yaml

```diff
@@ -135,6 +135,11 @@ merlin:
     # This should be the actual DNS registered
     trackingURL: "http://www.example.com"
 
+  database:
```

Comment:
Can you comment these out and add description such as what is it and when is it being used.

Reply:
Done